CONTRIBUTORS.txt

```diff
@@ -63,9 +73,11 @@ needs to be listed here.
 - Jason Toffaletti <jtoffaletti@apple.com>
 - Jim Dovey <jimdovey@mac.com>
 - Joachim M. <joachim.me86@gmail.com>
+- Joakim Hassila <hassila@users.noreply.github.com>
```

Comment:
@hassila  if you would like to have a real email address here, please add a comment with your preferred address. Thank you!

Reply:
Please use jocke@ordo.one  - thanks!